### PR TITLE
Contact default

### PR DESCRIPTION
**Pull request by tuancandongsang**

### CODE REVIEW

File: src/lib/components/UI/InputCurrency.svelte [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	export let placeholder = '0.00';
 </script>
 
-<label for="price" class="block text-sm font-medium text-gray-700">{label}</label>
+<label for="price" class="block text-sm font-medium text-gray-700">{label}ffffffffffff</label>
```
Comment (9a24f0) [inline src/lib/components/UI/InputCurrency.svelte]: What is this?

File: src/lib/components/UI/InputNumber.svelte [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	export let step = 1;
 </script>
 
-<label for="price" class="block text-sm font-medium text-gray-700">{label}</label>
+<label for="price" class="block text-sm font-medium text-gray-700">{label}sssssssssss</label>
```
Comment (9a24f0) [inline src/lib/components/UI/InputNumber.svelte]: Also this?

File: src/lib/components/UI/PrimaryButton.svelte [hunk 3]
```diff
@@ -8,20 +8,7 @@
...
-				? 'px-3 py-2 text-sm leading-4 font-medium'
-				: buttonSize === 'md'
-				? 'px-4 py-2 text-sm font-medium'
-				: buttonSize === 'lg'
-				? 'px-4 py-2 text-base font-medium'
-				: buttonSize === 'xl'
-				? 'px-6 py-3 text-base font-medium'
-				: ''
-		}inline-flex items-center border border-transparent rounded-md shadow-sm text-white bg-cyan-600 hover:bg-cyan-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-cyan-500`}
+		class="w-40 px-9 py-3 bg-green-500 rounded-3xl text-sm font-semibold text-white cursor-pointer hover:opacity-80"
```
Comment (9a24f0) [inline src/lib/components/UI/PrimaryButton.svelte]: We should keep conditionally rendering button like previous version. Please revert to older one and add styling base on the case we need.

File: src/lib/components/contact/ContactDefault.svelte [hunk 4]
```diff
@@ -55,7 +55,7 @@
 				rows="4"
 				name="message"
 				id="message"
-				class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full border-gray-300 rounded-md"
+				class="mt-1 focus:ring-cyan-500 focus:border-cyan-500 block w-full shadow-sm border-darkblue rounded-md"
```
Comment (9a24f0) [inline src/lib/components/contact/ContactDefault.svelte]: Nice shadow catch!

File: src/lib/components/contact/ContactMain.svelte [hunk 5]
```diff
@@ -5,22 +5,16 @@
 	export let currentFormChild;
 </script>
 
-<nav class="font-medium flex flex-col gap-6 text-[2.8125rem] font-poppins text-white">
-	<button
-		class="self-center transition ease-in-out duration-300 hover:translate-y-2"
-		on:click={() => contactScreen.set('general')}>
+<nav
+	class="font-medium not-italic semi-bold flex flex-col gap-6 text-[45px] font-poppins text-white"
```
Comment (9a24f0) [inline src/lib/components/contact/ContactMain.svelte]: `<nav>` tag has only one prop is class, please have them inline with the tag

File: src/lib/components/contact/ContractNavigation.svelte [hunk 6]
```diff
@@ -1,12 +1,28 @@
...
+	>
+	{#each arr as item}
+		<button
+			class="cursor-pointer my-1.5 block font-normal text-sm text-darkblue"
+			on:click={() => changeStateContact(item)}
+		>
+			{item}
+		</button>
+	{/each}
+</nav>
```
Comment (9a24f0) [inline src/lib/components/contact/ContractNavigation.svelte]: I don't quite understand this. Please elaborate

File: src/routes/contact/+page.svelte [hunk 7]
```diff
@@ -14,16 +15,20 @@
...
+      {#if $contactScreen === 'general'}
+        <ContactDefault/>
+      {/if}
+      {#if $contactScreen === 'entrepreneur'}
+        <ContactEntrepreneur/>
+      {/if}
+      {#if $contactScreen === 'partner'}
+        <ContactPartner/>
+      {/if}
+     </div>
```
Comment (9a24f0) [inline src/routes/contact/+page.svelte]: We use tab spacing, please convert those spaces to tab

**Review by 9a24f0 — CHANGES_REQUESTED** [review]
Also please replace function `changeState` with actual state and store in `generalStore`

File: src/lib/components/UI/PrimaryButton.svelte (hunk 3 above)
Comment (9a24f0) [reply src/lib/components/UI/PrimaryButton.svelte]: should be like this:
```
class="{`${
			buttonSize === 'xs'
				? 'px-2.5 py-1.5 text-xs font-medium'
				: buttonSize === 'sm'
				? 'px-3 py-2 text-sm leading-4 font-medium'
				: buttonSize === 'md'
				? 'px-4 py-2 text-sm font-medium'
				: buttonSize === 'lg'
				? 'px-4 py-2 text-base font-medium'
				: buttonSize === 'xl'
				? 'px-6 py-3 text-base font-medium'
				: ''
		} w-40 px-9 py-3 bg-green-500 rounded-3xl text-sm font-semibold text-white cursor-pointer hover:opacity-80"
```

File: src/lib/components/UI/InputCurrency.svelte (hunk 1 above)
Comment (tuancandongsang) [reply src/lib/components/UI/InputCurrency.svelte]: oke

**Review by 9a24f0 — APPROVED** [review]
Thank you!

**Pull request merged**